apps/api/src/resources/swagger.yaml

```diff
@@ -223,31 +236,9 @@ components:
           type: number
           description: A ID referring to the entry with the application contents.
 
-    ApplicationCreation:
```

Comment:
Moved to /requests to separate between db model definitions and req body definitions